Okta/OktaAuth/OktaAppAuth.swift

```diff
@@ -10,67 +10,71 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import Hydra
-
 // Current version of the SDK
 let VERSION = "2.1.0"
 
 // Holds the browser session
 public var currentAuthorizationFlow: OIDExternalUserAgentSession?
 
 // Cache Okta.plist for reference
-public var configuration: [String: Any]?
+public var configuration = try? OktaAuthConfig.default()
 
 // Cache the Discovery Metadata
 public var wellKnown: [String: Any]?
```

Comment:
Very confusing name of the variable. How about to rename to discoveredMetadata?

Reply:
Done.